wpilibcExamples/src/main/cpp/examples/GearsBot/cpp/subsystems/Wrist.cpp

```diff
@@ -4,10 +4,16 @@
 
 #include "subsystems/Wrist.h"
 
+#include <frc/RobotBase.h>
 #include <frc/controller/PIDController.h>
 #include <frc/smartdashboard/SmartDashboard.h>
 
 Wrist::Wrist() : frc2::PIDSubsystem{frc::PIDController{kP, 0, 0}} {
+  if constexpr (frc::RobotBase::IsSimulation()) {
+    // Conversion value of potentiometer varies between the real world and
+    // simulation
+    m_pot = frc::AnalogPotentiometer{3};  // Defaults to meters
+  }
```

Comment:
Is there a better way of doing conditional initialization based on a constexpr variable, other than an immediately invoked lambda?

Reply:
Ternary would work too.